composer.json

```diff
@@ -125,6 +125,10 @@
         }
     },
     "repositories": {
+        "acquia_cms_starterkit": {
+            "type": "vcs",
+            "url": "git@github.com:acquia/acquia-cms-starterkit.git"
+        },
```

Comment:
This package exists on the main packagist:
https://packagist.org/packages/acquia/acquia-cms-starterkit

We should get the administration of that sorted out (e.g. it's not currently auto-updated). This repo definition should be removed.

Reply:
Done.